src/Config/Level/CodeQualityLevel.php

```diff
@@ -175,6 +176,7 @@ final class CodeQualityLevel
         CompleteMissingIfElseBracketRector::class,
         RemoveUselessIsObjectCheckRector::class,
         StaticToSelfStaticMethodCallOnFinalClassRector::class,
+        FuncLikeToFirstClassCallableRector::class,
```

Comment:
same like `FunctionFirstClassCallableRector` https://getrector.com/rule-detail/function-first-class-callable-rector

it should not part of any set, as imo, it is by preference.